internal/daysteps/daysteps.go

```diff
@@ -10,6 +15,25 @@ var (
 
 func parsePackage(data string) (int, time.Duration, error) {
 	// ваш код ниже
+
+	parts := strings.Split(data, ",")
+	if len(parts) != 2 {
+		return 0, 0, fmt.Errorf("неверный формат данных")
+	}
+
+	countStep := parts[0]
+	count, err := strconv.Atoi(countStep)
+	if err != nil {
+		return 0, 0, fmt.Errorf("ошибка при парсинге количества")
```

Comment:
1. Теряешь сообщение об ошибки
2. Ошибку лучше писать на английском
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```

Reply:
Нужно везде поправить